gobblin-modules/gobblin-compliance/src/main/java/gobblin/compliance/ComplianceConfigurationKeys.java

```diff
@@ -33,6 +33,16 @@
   public static final String PARTITION_NAME = COMPLIANCE_PREFIX + ".partition.name";
   public static final int TIME_STAMP_LENGTH = 13;
   public static final String DBNAME_SEPARATOR = "__";
+  public static final String NUM_ROWS = "numRows";
+  public static final String RAW_DATA_SIZE = "rawDataSize";
```

Comment:
These are the property names inside the partition properties. So can't change it